flink-autoscaler/src/main/java/org/apache/flink/autoscaler/config/AutoScalerOptions.java

```diff
@@ -154,6 +154,15 @@ private static ConfigOptions.OptionBuilder autoScalerConfig(String key) {
                                     + RESTART_TIME.key()
                                     + "' will act as an upper bound.");
 
+    public static final ConfigOption<Duration> MAX_RESTART_TIME =
+            autoScalerConfig("restart.time.max")
+                    .durationType()
+                    .defaultValue(Duration.ofMinutes(30))
```

Comment:
We have internal overrides for all configuration defaults, but I don't think this overly pessimistic value is a good default for the general public. I would set this to not more than 15 minutes, even that is still conservative.

Reply:
Done.